app/src/main/res/values/strings.xml

```diff
@@ -142,9 +142,9 @@
   <string name="add_profile_error_pin_confirm_wrong">Please make sure that both PINs match.</string>
   <!-- PinPasswordActivity -->
   <string name="pin_password_hello">Hi, %s!</string>
-  <string name="pin_password_admin_enter">Please enter your \nAdministrator PIN</string>
-  <string name="pin_password_user_enter">Please enter your PIN</string>
-  <string name="pin_password_forgot_pin">I forgot my pin</string>
+  <string name="pin_password_admin_enter">Please enter your \nAdministrator PIN.</string>
```

Comment:
I am not sure if this `\n` should be used as it means that even on landscape mode it will contain 2 lines. Can you just confirm with @mschanteltc regarding this.

Reply:
@mschanteltc do we have this text `Please enter your Administrator PIN` in 2 lines in landscape mock or its a single line text in landscape. Please confirm.

Reply:
In landscape mode, the text will be one line. So the text in portrait mode should be bounded by a text box size. My mocks have it so that is is 240px wide.

Reply:
Thanks @mschanteltc made textbox size as 240px wide.

Reply:
To avoid giving fixed width. Created dimens file for different screen size, to have double line text in portrait mode and single line in landscape.